.github/workflows/testTKG.yml

```diff
@@ -2,6 +2,10 @@ name: "Run TKG Tests"
 on:
   issue_comment:
     types: [created]
+permissions:
+    contents: write
+    pull-requests: write
+    issues: write
 jobs:
```

Comment:
I think we will have to do this in a separate PR, merge it and then you will get to resume the testing you want to do.